src/core/database/database.service.ts

```diff
@@ -1278,25 +1302,7 @@ export class DatabaseService {
             }),
           ],
           ...properties,
-          [
-            node('adminSG', 'SecurityGroup', {
-              active: true,
-              createdAt,
-              name: sgName + ' admin',
-            }),
-            relation('out', '', 'member', { active: true, createdAt }),
-            node('requestingUser'),
-          ],
-          [
-            node('readerSG', 'SecurityGroup', {
-              active: true,
-              createdAt,
-              name: sgName + ' users',
-            }),
-            relation('out', '', 'member', { active: true, createdAt }),
-            node('requestingUser'),
-          ],
-          ...permissions,
+          ...permissionQueries,
```

Comment:
This does not work. You still need to create the security group and the permission nodes.  This removes them entirely.  A node that a root user creates still operates like a normal node.  
It is the matchSession call
``` 
     const query = this.db
        .query()
        .match(matchSession(session, { withAclEdit: aclEditPropName }))
```
that cannot fail when the user is a root user.  If this is already true, then leave it alone.

Reply:
Created RSG and Permission nodes.